core/templates/dev/head/components/button-directives/hint-and-solution-buttons.directive.html

```diff
@@ -73,6 +73,9 @@
     margin-top: 5px;
     padding: inherit;
     width: 32px;
+    min-width: 22px ;
```

Comment:
Here and elsewhere, please follow the existing style of the codebase.

Reply:
Done, Updated Thanks!